Assets/Scripts/Runtime/test.cs

```diff
@@ -4,15 +4,11 @@
 
 public class test : MonoBehaviour
 {
+    private string _hoge = "Hello, World!!!";
```

Comment:
いいね！

- [ ] 確認

@i-okumura 
#4 　テスト
```suggestion
    private string _hoge = "Hello, World!!!!!!!!!!!1";
```

Reply:
### なるほど

Reply:
うん？